packages/x-adapter-platform/src/mappers/responses/__tests__/__snapshots__/search-response.mapper.spec.ts.snap

```diff
@@ -409,7 +392,7 @@ Object {
       "image": "https://assets.empathy.co/images-demo/31335.jpg",
       "modelName": "Promoted",
       "tagging": Object {
-        "query": Object {
+        "click": Object {
```

Comment:
I would say that the tagging&click object of the promoted should be equal to the banner one